src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md

```diff
@@ -156,7 +152,7 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- ENGCOM-6040-->
 
-*  Magento no longer disables the cache when you run `composer update`. *Fix submitted by adrian-martinez-interactiv4 in pull request [24892](https://github.com/magento/magento2/pull/24892)*. [GitHub-17634](https://github.com/magento/magento2/issues/17634)
+*  In rare circumstances, executing `composer update` disabled all Magento caches, this is now resolved. *Fix submitted by adrian-martinez-interactiv4 in pull request [24892](https://github.com/magento/magento2/pull/24892)*. [GitHub-17634](https://github.com/magento/magento2/issues/17634)
```

Comment:
The description previously seemed to suggest this issue occurred all the time, but that wasn't true, it only occurred very irregularly.

Reply:
Reworded and added to .md files.